tests/unit/workspace_access/test_generic.py

```diff
@@ -814,3 +814,116 @@ def perms(object_type: str, object_id: str):
             tasks.append(task)
     assert len(tasks) == 1
     assert tasks[0].object_type == "clusters"
+
+
+def test_verify_task_should_return_true_if_permissions_applied():
+    ws = MagicMock()
```

Comment:
could you use `create_autospec(WorkspaceClient)` instead? it's "safer"

Reply:
done